www/googlemaps-cdv-plugin.js

```diff
@@ -1445,21 +1446,41 @@ Marker.prototype.remove = function(callback) {
 };
 
 App.prototype.removeMultipleMarkers = function(markers, callback) {
-    markerIds = [];
+    var self = this;
+    var markerIds = [];
 
     markers.forEach( function(marker){
       delete MARKERS[marker.id];
       markerIds.push(marker.id);
     })
     cordova.exec(function() {
         if (typeof callback === "function") {
-            callback.call(markerIds);
+            callback.call(self, markers, self);
         }
     }, this.errorHandler, PLUGIN_NAME, 'exec', ['Marker.removeMultipleMarkers', markerIds]);
 
     markers.forEach( function(marker){ marker.off(true); });
 };
 
+App.prototype.removeMultipleMarkersByType = function(type, callback) {
+    var self = this;
```

Comment:
It's fine, code doesn't need to be perfect, it just needs to work and to be readable enough, so we're good here! 🙂 

(It's just that it hurts my eyes a little to see `var self = this;` 🙈)